<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>from</class>
 <widget class="QWidget" name="from">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>154</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>535</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>35</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_temp">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-27 C</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_city">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_type">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>63</y>
     <width>175</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>175</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>type weather</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_fell">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>83</y>
     <width>123</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>123</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>123</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>0 C</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_pres">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>83</y>
     <width>123</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>123</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>123</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>750</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_wind">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>83</y>
     <width>123</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>123</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>123</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3 м/с</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp_1">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>105</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ощущается</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>105</y>
     <width>122</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>122</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>122</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>давление мм.р.ст.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_temp_3">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>105</y>
     <width>116</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>скорость ветра</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>123</y>
     <width>378</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>378</width>
     <height>33</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>378</width>
     <height>33</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bookman Old Style</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>true</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(198, 198, 198);
border-color: rgb(160, 160, 160);</string>
   </property>
   <property name="text">
    <string>_____</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_day">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>132</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>воскресеьне</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_logo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>8</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>126</y>
     <width>371</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>384</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="box">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
   </layout>
  </widget>
  <zorder>l_temp</zorder>
  <zorder>l_city</zorder>
  <zorder>l_type</zorder>
  <zorder>l_fell</zorder>
  <zorder>l_pres</zorder>
  <zorder>l_wind</zorder>
  <zorder>l_temp_1</zorder>
  <zorder>l_temp_2</zorder>
  <zorder>l_temp_3</zorder>
  <zorder>l_logo</zorder>
  <zorder>line</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>btn_more</zorder>
  <zorder>l_day</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
